lib/takeaway.rb

```diff
@@ -0,0 +1,30 @@
+require_relative './menu.rb'
+require_relative './order.rb'
+
+class TakeAway
+
+  attr_reader :order, :menu
+
+  def initialize
+    @menu = Menu.new
+  #  @order = order
+  end
+
+  # def order
+  #   @order
+  # end
+
+  def new_order
+    @order = Order.new
+    @menu.print_menu
```

Comment:
Nice use of [forwarding](https://github.com/makersacademy/skills-workshops/blob/master/practicals/object_oriented_design/oo_relationships.md#forwarding)!